hack/gotest.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+CGO_ENABLED=0
+# in prow, already in container, so no 'docker run'
+if [ "$IS_CONTAINER" != "" ]; then
+  cd installer/pkg
+  PKGS=$(go list ./...)
+  for p in $PKGS; do
+    go test "$p" -cover
```

Comment:
We want to fail this scrip if any of these fail, and a `for` loop makes that harder.  I think you can just do:

````sh
if [ "$IS_CONTAINER" != "" ]; then
  CGO_ENABLED=0 go test -cover ./installer/...
else
  docker ...
fi
```

Reply:
yes, you are right. fixed